moped-editor/netlify.toml

```diff
@@ -5,7 +5,7 @@
 #   From: https://docs.netlify.com/site-deploys/overview/#deploy-contexts
 #   Also: https://docs.netlify.com/configure-builds/environment-variables/
 [context.branch-deploy]
-    command = "echo 'Build Context Mode: Branch-Deploy' && npm run build:netlifypr"
+    command = "echo 'Build Context Mode: Branch-Deploy' && npm run build:netlifybranch"
```

Comment:
This is a subtle crux of the whole PR.

@mddilley - I'd really enjoy getting to sync up with you to go over the changes I'm proposing here and to get your feedback. Maybe we can do that in the next few days?

Reply:
The group's response to [this question in slack](https://austininnovation.slack.com/archives/CNUEPKLB1/p1658693048535269) will matter a lot in terms of the viability of this PR. If there is a need to keep auto build & deploy turned on for all branch pushes, this PR may need to change how it initiates test deployment frontend builds.

Reply:
@frankhereford That sounds great to me! I think syncing up tomorrow would be great to catch each other up on the progress. I'll take a look at this PR in the morning and see when you have some free time. 🚀